Library/Homebrew/dev-cmd/audit.rb

```diff
@@ -121,8 +123,18 @@ def audit
     end
 
     return if problem_count.zero?
+    errors_summary = "#{Formatter.pluralize(problem_count, "problem")} in #{Formatter.pluralize(formula_count, "formula")}"
+    if new_formula && new_formula_problem_lines.size.zero?
+      GitHub.create_issue_comment("repository-name", "pull-request-number",
```

Comment:
I think we should pick a `HOMEBREW_` environment variable for these values which can be passed in by `brew test-bot`. I think if they are missing or the API call fails it'd be good to just output them as usual. Thoughts?

Reply:
@GauthamGoli I think if the environment variables are set: yes.

Reply:
@MikeMcQuaid  Updated the PR accordingly. Let me know what you think.